evaluators/embedding/MinkowskiDistanceEvaluator/__init__.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class MinkowskiDistanceEvaluator(BaseEmbeddingEvaluator):
+    """A :class:`MinkowskiDistanceEvaluator` evaluates the distance between actual and desired 
+    embeddings computing the Minkowski distance ( p>0 ) between them, which can be considered 
+    as a generalization of both the Euclidean distance and the Manhattan distance.
+
+        D(x, y) = ( sum_i |x_i - y_i|^p )^(1/p)
+    """
+
+    @property
+    def metric(self):
+        return 'MinkowskiDistanceEvaluator'
+
+    def evaluate(self, actual: 'np.array', desired: 'np.array', *args, **kwargs) -> float:
+        """"
+        :param actual: the embedding of the document (resulting from an Encoder)
+        :param desired: the expected embedding of the document
+        :return the evaluation metric value for the request document
+        """
+        actual = expand_vector(actual)
+        desired = expand_vector(desired)
+        order = kwargs["order"]
+        return _minkowski_Distance(actual, desired, order)
+
+
+def _minkowski_Distance(actual, desired, order):
```

Comment:
We tend to use underscore case, please do not mix it with camelCase.

Reply:
thanks for all your nice suggestions.